apps/web/src/components/pet/PetPage.module.css

```
@@ -279,11 +548,160 @@
+  text-shadow: 0 0 12px rgba(116, 247, 161, 0.35);
+}
+
+.trainingFeedbackText {
+  margin: 0;
+  color: #fff8d7;
+  font-family: "DotGothic16", monospace;
+  font-size: clamp(0.82rem, 1.12vw, 0.94rem);
+  line-height: 1.45;
+}
+
+@keyframes trainingToast {
```

Stylelint (17.11.1)

[error] Expected keyframe name "trainingToast" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)